frontend/src/components/nodes/NodeNav.tsx

```diff
@@ -58,11 +58,7 @@ export default class extends React.Component<State> {
                   src={"/static/images/icon-m-node-validator-status.svg"}
                   style={{ width: "12px", marginRight: "10px" }}
                 />
-                {validators ? (
-                  `${validators}  VALIDATOR NODES`
-                ) : (
-                  <Spinner animation="border" variant="secondary" />
-                )}
+                {validatorsCount ? `${validatorsCount}  VALIDATOR NODES` : `-`}
```

Comment:
I meant `- VALIDATOR NODES`.

Also, given our today's sync, let's call the tabs as "Validating" | "Non-Validating"